crates/core/plugin_sm/src/plugin_manager.rs

```diff
@@ -245,7 +245,7 @@ impl ExternalPlugins {
             }
         }
 
-        if let Some(reason) = ExternalPlugins::error_message(error_count, command_log) {
+        if let Some(reason) = ExternalPlugins::error_message(error_count, None, command_log) {
```

Comment:
Passing hard-coded `None` here defeats the purpose, right? The error that's ignored in line no. 241 should have been passed here, right?

Reply:
True. That's an artifact of me trying to solve the install case specifically, but you're right that we shouldn't discard the error.